aeron-driver/src/main/c/uri/aeron_driver_uri.c

```diff
@@ -48,6 +48,38 @@ int aeron_uri_get_term_length_param(aeron_uri_params_t *uri_params, aeron_driver
     return 0;
 }
 
+int aeron_uri_get_retransmits_active_max_param(aeron_uri_params_t *uri_params, aeron_driver_uri_publication_params_t *params)
+{
+    const char *value_str;
+
+    if ((value_str = aeron_uri_find_param_value(uri_params, AERON_URI_RETRANSMITS_ACTIVE_MAX_KEY)) != NULL)
+    {
+        uint64_t value;
+
+        if (-1 == aeron_parse_size64(value_str, &value))
+        {
+            AERON_SET_ERR(EINVAL, "could not parse %s=%s in URI", AERON_URI_RETRANSMITS_ACTIVE_MAX_KEY, value_str);
+            return -1;
+        }
+
+        if (value < 1 || value > AERON_RETRANSMIT_HANDLER_RETRANSMITS_ACTIVE_MAX_MAX)
+        {
+            AERON_SET_ERR(
+                EINVAL,
+                "invalid %s=%llu, must be > 0 and <= %i",
```

Comment:
We should use the [standardised fixed width integer type](https://en.cppreference.com/w/c/types/integer) macros for printing, e.g. in this case PRIu64.